components/authz-service/storage/postgres/datamigration/sql/01_v2_data_migrations.up.sql

```diff
@@ -1,7 +1,6 @@
 BEGIN;
 
 SET CONSTRAINTS iam_policy_members_policy_id_fkey DEFERRED;
-SET CONSTRAINTS iam_policy_statements_policy_id_fkey DEFERRED;
```

Comment:
These are from the upgrade migrations-- not to be confused with those with which we cannot tamper